src/app/modules/submission/encrypt-submission/__tests__/encrypt-submission.controller.spec.ts

```diff
@@ -292,4 +314,219 @@ describe('encrypt-submission.controller', () => {
       ).toHaveBeenCalledWith(MOCK_FORM_ID, mockReq.query.page)
     })
   })
+
+  describe('handleStreamEncryptedResponses', () => {
+    const MOCK_USER_ID = new ObjectId().toHexString()
+    const MOCK_FORM_ID = new ObjectId().toHexString()
+    const MOCK_USER = {
+      _id: MOCK_USER_ID,
+      email: 'somerandom@example.com',
+    } as IPopulatedUser
+    const MOCK_FORM = {
+      admin: MOCK_USER,
+      _id: MOCK_FORM_ID,
+      title: 'mock title',
+    } as IPopulatedForm
+
+    const MOCK_REQ = expressHandler.mockRequest({
+      params: {
+        formId: MOCK_FORM_ID,
+      },
+      session: {
+        user: {
+          _id: MOCK_USER_ID,
+        },
+      },
+    })
+
+    // Not sure how to test streams in express controllers, so skipping...
+    it.todo('should successfully return stream of encrypted responses')
```

Comment:
is this a good time to add an integration test for this endpoint?

Reply:
i'll do it in a later PR when I migrate the routes to Typescript